README.md

```diff
@@ -17,20 +18,22 @@ apt install git python3-dev python3-pip
 
 Clone repository
 ```sh
-git clone git@github.com:kevinjycui/DesmosBezierRenderer.git
```

Comment:
SSH cloning is fine, this doesn't have to be changed to HTTPS

Reply:
While that is true, https cloning is significantly easier as there are no prompts or anything. ![](https://xenon.is-ne.at/WfkiRx.png)